charts/tidb-operator/templates/admission/admission-webhook-registration.yaml

```diff
@@ -130,11 +101,11 @@ webhooks:
         resources: ["statefulsets"]
 {{- end }}
 ---
-{{- if .Values.admissionWebhook.hooksEnabled.validating }}
+{{- if .Values.admissionWebhook.validation.pingcapResources }}
 apiVersion: admissionregistration.k8s.io/v1beta1
 kind: ValidatingWebhookConfiguration
 metadata:
-  name: pingcap-resources-validating
+  name: pingcap-tidb-resources-validating
```

Comment:
As ChaosMesh released, tidbcluster is not the only resources under pingcap group.